views/default/js/mentions/config.js.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Provides information which editor plugin is used on the site
+ *
+ * Checks which editor plugin (if any) is enabled and provides
+ * the information as an AMD module.
+ */
+
+$plugins = array(
+	'ckeditor',
+	'tinymce',
```

Comment:
This won't support plugins like "tinymce_extended", but we can figure out that later. Perhaps use a plugin hook.